browser/ui/views/brave_actions/brave_rewards_action_stub_view.cc

```diff
@@ -44,10 +44,13 @@ BraveRewardsActionStubView::BraveRewardsActionStubView(
   std::unique_ptr<IconWithBadgeImageSource> image_source(
       new BraveActionIconWithBadgeImageSource(preferred_size));
   // Set icon on badge using actual extension icon resource
-  const auto image = gfx::Image(
-      rb.GetImageNamed(IDR_BRAVE_REWARDS_ICON_64).AsImageSkia()
-          .DeepCopy());
-  image_source->SetIcon(image);
+  gfx::ImageSkia image;
+  const SkBitmap bitmap = rb.GetImageNamed(IDR_BRAVE_REWARDS_ICON_64)
```

Comment:
Should include `third_party/skia/include/core/SkBitmap.h`?

Reply:
Thanks, guess I should take a break! 🤦‍♂